sdk/src/manifest_store.rs

```diff
@@ -267,14 +267,15 @@ impl ManifestStore {
     /// ```
     pub async fn from_manifest_and_asset_bytes_async(
```

Comment:
This is a public breaking API change so we need to make sure we document it

Reply:
👍